test/componentSetup.js

```diff
@@ -0,0 +1,17 @@
+const userEvent = require('@testing-library/user-event').default
+const { render } = require('@testing-library/react')
```

Comment:
can use imports here or no?

Reply:
it wasn't working straight away and I didn't feel like messing with the jest test setup